### PR TITLE
Implement OpenStack SecurityGroup checker plugin

### PR DESCRIPTION
**Pull request by redixin**

### CODE REVIEW

File: security/plugins/secgroup.py [hunk 1]
```diff
@@ -13,17 +13,61 @@
...
+
+
+def match_rule(pattern, rule):
+    for field, value in pattern.items():
+        if isinstance(value, tuple):
+            if rule[field] not in value:
+                return None
+        else:
+            if rule[field] != value:
+                return None
```
Comment (teferi) [inline security/plugins/secgroup.py]: nit:`False` feels a bit more coherent.

Side note: I've spent a couple of minutes going through all the ifs, to see what the function actually does. However I don't have a good alternative for it =)

Comment (redixin) [reply security/plugins/secgroup.py]: > False feels a bit more coherent.

Yes. At first I was going to return None or Issue instance, but its easier to create Issue out of  this function.

**Review by teferi — APPROVED** [review]
lgtm

**Review by boris-42 — APPROVED** [review]
okay let's start

**Pull request merged**